### DISCUSSION TITLE
Is there a way to set that it starts minimized to tray?

### DISCUSSION THREAD

**Discussion by ghost** [wwmm/easyeffects#1017] [Q&A]
I want that it starts minimized to tray on autostart.

**Comment by wwmm** [comment]
There isn't. gtk4 does not support tray icons. When EasyEffects is started as a service `easyeffects --gapplication-service` it will keep running in the background even if you close its window. What is usually most people intention when trying to minimize to tray. There is no need to keep our window open in order to apply effects.

**Reply by kubrickfr** [reply]
So what is the easy way to shut it down? (by easy I mean, no `easyeffects -q`)

**Reply by wwmm** [reply]
The shortcut `Ctrl+Q`.

**Reply by kubrickfr** [reply]
This requires the application to be opened and in focus though

**Reply by hboetes** [reply]
create a keybind that runs the command `pkill easyeffects`

**Reply by kubrickfr** [reply]
I'll never understand gnome/redhat for having dropped tray-icons support...

**Comment by hboetes** [comment]
I just created a startup script like this:

```bash
#!/bin/sh
if ! pgrep -f /usr/bin/easyeffects; then
    exec /usr/bin/easyeffects --gapplication-service &
    notify-send 'easyeffects daemon started'
else
    /usr/bin/easyeffects
fi
```

So it starts `easyeffects` in the background if it isn't running and opens the interface if it is.
